### ISSUE TITLE
create a transfer money

### ISSUE THREAD

**Issue by ChaerilM** [leMaik/moneylover-cli#5]
is there a way to transfer money between 2 wallets?, like from bank to cash in hand or credit cards.

**Comment by leMaik** [comment]
Good idea, that would be helpful. :+1: In the app the transfer seems to work like this:

* A "Other" transaction for the wallet that sends the money ("Send to: <other wallet name>")
* A "Gift" transaction for the wallet that receives the money ("Received from: <other wallet name>")

Should be simple. The first transaction might include a transaction fee (will be stored as a third transaction) and all the transactions are excluded from the report by default.
